plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -197,9 +179,12 @@ class NewCreateServiceModalForm extends Component {
   }
 
   getAppConfig(currentState = this.state) {
-    let {baseConfig, batch} = currentState;
-    let patch = batch.reduce(jsonConfigReducers, {});
-    return CreateServiceModalFormUtil.applyPatch(baseConfig, patch);
+    let {batch} = currentState;
+    return batch.reduce(this.props.jsonConfigReducers, {});
```

Comment:
⚠️ 🚨 ⚠️ 🚨 ⚠️ This will cause more problems than solving!

```js
let {baseConfig, batch} = currentState;
delete baseConfig.env; // << Do this for all the dynamic fields
let patch = batch.reduce(jsonConfigReducers, {});
return CreateServiceModalFormUtil.applyPatch(baseConfig, patch);
```